### DISCUSSION TITLE
blog/welcome-to-jekyll/

### DISCUSSION THREAD

**Discussion by giscus** [msindev/penguincoders.net#5] [Blog Comments]
# blog/welcome-to-jekyll/

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated.

https://penguincoders.net/blog/welcome-to-jekyll/

**Comment by msindev** [comment]
Test comment
